contracts/bridges/SweepableBridgeReceiver.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../IERC20NonStandard.sol";
+import "./BridgeReceiverCore.sol";
+
+contract SweepableBridgeReceiver is BridgeReceiverCore {
```

Comment:
What about having the `SweepableBridgeReceiver` extend `BaseBridgeReceiver` directly? That way, the `ArbitrumBridgeReceiver` can just extend `SweepableBridgeReceiver` instead of both `BaseBridgerReceiver, SweepableBridgeReceiver`, which is a bit confusing IMO.

Reply:
was just thinking something similar, it's an extra contract for no particular reason